api/src/certification/configuration/infrastructure/serializers/certification-version-serializer.js

```diff
@@ -0,0 +1,18 @@
+import { Version } from '../../domain/models/Version.js';
+
+const deserialize = (json) => {
+  const attributes = json.data.attributes;
+
+  return new Version({
+    id: json.data.id ? parseInt(json.data.id) : undefined,
```

Comment:
```suggestion
    id: json.data.id
```
Question: La paramètre `id` est mentionné comme required dans le payload de la route, pour quelle raison ne serait-il pas présent ?